MC/config/PWGHF/pythia8/generator/pythia8_charmhadronic_with_decays_Mode2_CorrBkg.cfg

```
@@ -185,33 +185,35 @@ BeamRemnants:saturation 5
 411:onIfMatch = 10311 321
 ### D+/- -> φ π
 411:onIfMatch = 333 211
-### D+/- -> π π π
-411:onIfMatch = 211 211 211
+### D+/- -> K K π 
+411:onIfMatch = 321 321 211
 ### D+/- -> f2(1270) π
 411:onIfMatch = 225 211
 ### D+/- -> rho π
 411:onIfMatch = 113 211
+### D+/- -> π π π
+411:onIfMatch = 211 211 211
 
 ### Ds -> φ π
 431:onIfMatch = 333 211
 ### Ds -> K* K
 431:onIfMatch = 313 321
 ### Ds -> φ rho+
 431:onIfMatch = 333 213
-### Ds -> π π π
-431:onIfMatch = 211 211 211
-### Ds -> f2 π
-431:onIfMatch = 225 211
 ### Ds -> rho π
 431:onIfMatch = 113 211
-### Ds -> K π π
-431:onIfMatch = 321 211 211
+### Ds -> f2 π
+431:onIfMatch = 225 211
+### Ds -> π π π
+431:onIfMatch = 211 211 211
 ### Ds -> K*(892)0 π+
 431:onIfMatch = 313 211
 ### Ds -> f0(1370) K+
 431:onIfMatch = 10221 321
 ### Ds -> rho0 K+
 431:onIfMatch = 113 321
+### Ds -> K π π
+431:onIfMatch =  211 321 211
```

```suggestion
431:onIfMatch =  321 211 211
```
I'd keep the order as the one defined at line 63